frontend/src/components/DocumentViewer/DocumentViewBox.test.tsx

```diff
@@ -261,4 +261,21 @@ describe('DocumentViewBox component tests', () => {
 			expect(nextButton).toBeEnabled();
 		});
 	});
+
+	describe('With zero documents', () => {
+		const documents = [] as MockDocument[];
+
+		beforeEach(() => {
+			setupMocks(documents);
+		});
+
+		test('GIVEN are zero documents THEN an error message is shown', async () => {
+			const ExpectedErrorText =
+				'Unable to fetch documents. Try again in a few minutes.';
+			renderDocumentViewBox();
+			const messageElement = await screen.findByText(ExpectedErrorText);
+
+			expect(messageElement).toBeInTheDocument();
```

Comment:
Do we care that the document navigator is not visible, or just that the error message is visible? Probably the latter, so I think this is fine.

I think the test name is missing a "there" after GIVEN.